components/autofill/src/autofill.udl

```diff
@@ -80,53 +80,55 @@ dictionary Address {
 };
 
 [Error]
-enum AutofillError {
-   "OpenDatabaseError", "SqlError", "IoError", "InterruptedError",
-   "IllegalDatabasePath", "Utf8Error", "JsonError", "InvalidSyncPayload",
-   "MissingEncryptionKey", "CryptoError", "NoSuchRecord",
+interface AutofillApiError {
+    SqlError(string reason);
+    InterruptedError();
+    CryptoError(string reason);
+    NoSuchRecord(string guid);
```

Comment:
Should all error variants end in "Error"?  It seems maybe more idomatic, but I'm not sure.

Reply:
Let's just leave this alone then, it seems like it's not important and there's no clear right answer.